kafka.go

```diff
@@ -1,161 +1,213 @@
-/*
- * Copyright 2019 AccelByte Inc
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- * http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-package eventpublisher
+// Copyright (c) 2019 AccelByte Inc. All Rights Reserved.
```

Comment:
Not really sure about this, but should we use apache 2.0 license short header?

Reply:
sure, I will update it
thanks